src/documents/ChangeApplierAPI.md

```diff
@@ -248,7 +367,7 @@ at the time the component is constructed. Note that programmatically attached li
 The listener is notified after the change (or set of coordinated changes) has already been applied to the model - it is too late to affect this process and so this event is not _preventable_. The signature for these listeners is
```

Comment:
I didn't suspect that this seemingly harmless aside could cause such confusion - I've simply removed it :) The listeners are all perfectly normal and behave just the way you expect, both declaratively and programmatically. All I was trying to do was link these semantics to the ones for different kinds of events - and pointing out that model events behave like "standard" events rather than "preventable" ones.